FizzBuzz/Program.cs

```diff
@@ -1,12 +1,163 @@
 ﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace FizzBuzz
 {
+    public class FizzBuzz : IEnumerable<string>
+    {
+        private List<string> items;

```

Comment:
You didn't actually implement `IEnumerator`, right? You just used the implementation from `List`